### PR TITLE
add new hosts for eic/clas12 (SOFTWARE-5176)

### PR DESCRIPTION
**Pull request by edquist**
For SOFTWARE-5176, this adds the following hosts:

- fejlab.svc.opensciencegrid.org
- clas12.voms2.opensciencegrid.org
- eic.voms2.opensciencegrid.org

Note: I had to guess the port for all of these, and I guessed the VO name for fejlab.  (Don't know if this is correct!)

### CODE REVIEW

File: vomses [hunk 1]
```diff
@@ -52,6 +52,9 @@
...
 "clas12" "clas12.voms.opensciencegrid.org" "15001" "/DC=org/DC=incommon/C=US/ST=Wisconsin/L=Madison/O=University of Wisconsin-Madison/OU=OCIS/CN=clas12.voms.opensciencegrid.org" "clas12"
+"clas12" "clas12.voms2.opensciencegrid.org" "15001" "/DC=org/DC=incommon/C=US/ST=Wisconsin/O=University of Wisconsin-Madison/OU=OCIS/CN=clas12.voms2.opensciencegrid.org" "clas12"
 "wlcg" "wlcg-voms.cloud.cnaf.infn.it" "15001" "/DC=org/DC=terena/DC=tcs/C=IT/ST=Roma/O=Istituto Nazionale di Fisica Nucleare - INFN/CN=wlcg-voms.cloud.cnaf.infn.it" "wlcg"
 "eic" "eic.voms.opensciencegrid.org" "15001" "/DC=org/DC=incommon/C=US/ST=Wisconsin/L=Madison/O=University of Wisconsin-Madison/OU=OCIS/CN=eic.voms.opensciencegrid.org" "eic"
+"eic" "eic.voms2.opensciencegrid.org" "15001" "/DC=org/DC=incommon/C=US/ST=Wisconsin/O=University of Wisconsin-Madison/OU=OCIS/CN=eic.voms2.opensciencegrid.org" "eic"
 "kagra" "voms.cc.kek.jp" "15027" "/C=JP/O=KEK/OU=CRC/CN=host/voms.cc.kek.jp" "kagra"
+"fejlab" "fejlab.svc.opensciencegrid.org" "15001" "/DC=org/DC=incommon/C=US/ST=Wisconsin/O=University of Wisconsin-Madison/OU=OCIS/CN=fejlab.svc.opensciencegrid.org" "fejlab"
```
Comment (brianhlin) [inline vomses:60]:
```suggestion
```

File: vomsdir/fejlab/fejlab.svc.opensciencegrid.org.lsc [hunk 2]
```diff
@@ -0,0 +1,2 @@
+/DC=org/DC=incommon/C=US/ST=Wisconsin/O=University of Wisconsin-Madison/OU=OCIS/CN=fejlab.svc.opensciencegrid.org
+/C=US/O=Internet2/OU=InCommon/CN=InCommon IGTF Server CA
```
Comment (brianhlin) [inline vomsdir/fejlab/fejlab.svc.opensciencegrid.org.lsc:2]: This file/dir can be deleted

**Review by brianhlin — CHANGES_REQUESTED** [review]
Oops, it wasn't quite clear in the ticket: the clas12/eic needed to be added but the fejlab cert is the web cert for the JLab frontend and isn't used to sign VOMS attrs so it doesn't need to go into the `vo-client`

**Comment by edquist** [comment]
Ok, yeah i wasn't sure so i put the fejlab stuff in a separate commit; so i'm just backing up the PR one commit to remove it

File: vomsdir/fejlab/fejlab.svc.opensciencegrid.org.lsc (hunk 2 above)
Comment (edquist) [reply vomsdir/fejlab/fejlab.svc.opensciencegrid.org.lsc]: :+1:

File: vomses (hunk 1 above)
Comment (edquist) [reply vomses]: :+1:

**Review by brianhlin — APPROVED** [review]
LGTM!

**Pull request merged**